sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Release History
 
 ## 4.4.0-beta.1 (2020-08-27)
+### New Features
 * Added new API to efficiently load many documents (via list of pk/id pairs or all documents for a set ok pk values)
 * Enabled query metrics by default
-* Fixed NPE in GatewayAddressCache
+### Key Bug Fixes
+* Fixed NPE in `GatewayAddressCache`
```

Comment:
please put a "." at the end of each changelog entry
to be consistent with other changelog entries:

Reply:
Fixed